hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/security/token/OzoneBlockTokenIdentifier.java

```diff
@@ -170,6 +181,21 @@ public void write(DataOutput out) throws IOException {
     out.write(builder.build().toByteArray());
   }
 
+  @Override
+  public byte[] getBytes() {
+    BlockTokenSecretProto.Builder builder = BlockTokenSecretProto.newBuilder()
+        .setBlockId(blockId)
+        .setOwnerId(getOwnerId())
+        .setSecretKeyId(ProtobufUtils.toProtobuf(getSecretKeyId()))
+        .setExpiryDate(getExpiryDate())
+        .setMaxLength(maxLength);
+    // Add access mode allowed
+    for (AccessModeProto mode : modes) {
+      builder.addModes(AccessModeProto.valueOf(mode.name()));
+    }
+    return builder.build().toByteArray();
+  }
```

Comment:
`getBytes()` is overridden (optimized) only for block token.  Did you intend to do the same for container token?

Reply:
Good idea.

Also refactored OzoneBlockTokenIdentifier write() to use getBytes().